indigo_app/views/tasks.py

```diff
@@ -100,9 +100,29 @@ def get_context_data(self, **kwargs):
         # merge actions and comments
         actions = task.action_object_actions.all()
         task_content_type = ContentType.objects.get_for_model(self.model)
-        comments = Comment.objects\
+        comments = list(Comment.objects\
             .filter(content_type=task_content_type, object_pk=task.id)\
-            .select_related('user')
+            .select_related('user'))
+        
+        # get the annotation for the particular task
+        task_annotation = Annotation.objects.filter(task=task).first()
+
+        # for the annotation that is linked to the task, get all the replies
+        if task_annotation:
+            fake_comments = []
+            # get the replies to the annotation
+            annotation_replies = Annotation.objects.filter(in_reply_to=task_annotation)
+            for annotation in annotation_replies:
+                fake_comment = Comment(
+                    user=annotation.created_by_user,
+                    comment=annotation.text,
+                    object_pk=task_annotation.task,
```

Comment:
Is this line correct? Does the fake annotation really need an `object_pk` value and, if so, shouldn't it be an id field such as `task_annotation.task.pk`?

Reply:
I thought it is needed while rendering on the front end, it actually is not. Taking it off.